rome-osgi-test/pom.xml

```diff
@@ -11,6 +10,43 @@
     </parent>
 
     <artifactId>rome-osgi-test</artifactId>
+    <packaging>jar</packaging>
+
+    <name>rome-osgi-test</name>
+
+    <description>OSGi tests for ROME</description>
```

Comment:
What's the point of having `<name>` and `<description>` tags here when they don't give any additional information?